js/packages/teams-ai/src/models/index.ts

```diff
@@ -10,3 +10,4 @@ export * from './ChatCompletionAction';
 export * from './LlamaModel';
 export * from './OpenAIModel';
 export * from './PromptCompletionModel';
+export * from './TestModel';
```

Comment:
should `TestModel` be in the `internals` directory?

Reply:
I made it external so that developers could write unit tests for their bots. We could make it internal but then developers just have to re-implement the same basic logic to test their code.